src/ArduinoIoTCloud.cpp

```diff
@@ -276,6 +335,19 @@ void ArduinoIoTCloudClass::handleMessage(int length)
   if (_dataTopicIn == topic) {
     Thing.decode((uint8_t*)bytes, length);
   }
+  if ((_shadowTopicIn == topic) && _syncStatus == ArduinoIoTSynchronizationStatus::SYNC_STATUS_WAIT_FOR_CLOUD_VALUES) {
+    Thing.decode((uint8_t*)bytes, length, true);
+    sendPropertiesToCloud();
+    _syncStatus = ArduinoIoTSynchronizationStatus::SYNC_STATUS_VALUES_PROCESSED;
+  }
+}
+
+void ArduinoIoTCloudClass::requestLastValue()
+{
+  // Send the getLastValues CBOR message to the cloud
+  // [{0: "r:m", 3: "getLastValues"}] = 81 A2 00 63 72 3A 6D 03 6D 67 65 74 4C 61 73 74 56 61 6C 75 65 73
+  const uint8_t CBOR_REQUEST_LAST_VALUE_MSG[] =  { 0x81, 0xA2, 0x00, 0x63, 0x72, 0x3A, 0x6D, 0x03, 0x6D, 0x67, 0x65, 0x74, 0x4C, 0x61, 0x73, 0x74, 0x56, 0x61, 0x6C, 0x75, 0x65, 0x73 };
```

Comment:
Can we rewrite this message in a human friendly way? Is it never going to change again?

Reply:
Great, I think adding an instruction in the code on how to generate the bytes array could be enough

Reply:
Added a specific comment in the file.

Reply:
:+1: